contracts/peripheral/OhmBondManager.sol

```diff
@@ -0,0 +1,167 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity 0.8.15;
+
+import {IBondSDA} from "../interfaces/IBondSDA.sol";
+import {IBondTeller} from "../interfaces/IBondTeller.sol";
+import {IEasyAuction} from "../interfaces/IEasyAuction.sol";
+import {IERC20} from "../interfaces/IERC20.sol";
+import {ITreasury} from "../interfaces/ITreasury.sol";
+import {IOlympusAuthority} from "../interfaces/IOlympusAuthority.sol";
+import {OlympusAccessControlled} from "../types/OlympusAccessControlled.sol";
+
+contract OhmBondManager is OlympusAccessControlled {
+    // ========= DATA STRUCTURES ========= //
+    struct BondProtocolParameters {
+        address callbackAddress;
+        uint256 initialPrice;
+        uint256 minPrice;
+        uint32 debtBuffer;
+        uint256 auctionTime;
+        uint32 depositInterval;
+    }
+
+    struct GnosisAuctionParameters {
+        uint256 auctionCancelTime;
+        uint256 auctionTime;
+        uint96 minRatioSold;
+        uint256 minBuyAmount;
+        uint256 minFundingThreshold;
+    }
+
+    // ========= STATE VARIABLES ========= //
+
+    /// Tokens
+    IERC20 public ohm;
+
+    /// Contract Dependencies
+    ITreasury public treasury;
+
+    /// Market Creation Systems
+    IBondSDA public fixedExpiryAuctioneer;
+    IBondTeller public fixedExpiryTeller;
+    IEasyAuction public gnosisEasyAuction;
+
+    /// Market parameters
+    BondProtocolParameters public bondProtocolParameters;
+    GnosisAuctionParameters public gnosisAuctionParameters;
+
+    constructor(
+        address ohm_,
+        address treasury_,
+        address feAuctioneer_,
+        address feTeller_,
+        address gnosisAuction_,
+        address authority_
+    ) OlympusAccessControlled(IOlympusAuthority(authority_)) {
+        ohm = IERC20(ohm_);
+        treasury = ITreasury(treasury_);
+        fixedExpiryAuctioneer = IBondSDA(feAuctioneer_);
+        fixedExpiryTeller = IBondTeller(feTeller_);
+        gnosisEasyAuction = IEasyAuction(gnosisAuction_);
+    }
+
+    // ========= MARKET CREATION ========= //
+    function createBondProtocolMarket(uint256 capacity_, uint256 bondTerm_) external onlyPolicy returns (uint256) {
```

Comment:
I'm a little torn between providing a duration and a timestamp for the bondTerm. It would be easier to line up several bonds to use the same token if you used a timestamp since it's a little uncertain when MS transactions will go through. However, this is more likely to result in the outcome expected for issuing a single bond.

Reply:
hmm yeah that's a good point I hadn't thought about. I was just coming at it from the perspective we want to minimize potential truncation of bond term if the multisig takes a while to execute.